rotkehlchen/tests/api/test_misc.py

```diff
@@ -418,6 +420,20 @@ def test_cache_deletion(rotkehlchen_api_server):
     avatars_dir = icons_dir / 'avatars'
     avatars_dir.mkdir(exist_ok=True)
 
+    dbens = DBEns(rotkehlchen_api_server.rest_api.rotkehlchen.data.db)
+    with dbens.db.user_write() as write_cursor:
+        dbens.add_ens_mapping(
```

Comment:
why is this needed in the test?

Reply:
Querying an avatar that's not present in the ens_mappings will raise an error.

Reply:
then you should have added a comment explaining this is why you are doing it.